core/src/androidTest/java/novoda/lib/sqliteprovider/ContentProviderTest.java

```diff
@@ -1,12 +1,22 @@
 
 package novoda.lib.sqliteprovider;
 
+import android.content.ContentValues;
 import android.database.Cursor;
+import android.database.sqlite.SQLiteDatabase;
 import android.net.Uri;
 import android.test.AndroidTestCase;
 
+import java.io.IOException;
+
+import novoda.lib.sqliteprovider.sqlite.ExtendedSQLiteOpenHelper;
+
 public class ContentProviderTest extends AndroidTestCase {
 
+    private static final String TABLE_NAME = "test";
+    private static final String COLUMN_PRIMARY_KEY = "column_primary_key";
+    private static final String COLUMN2 = "column2";
```

Comment:
Why didn't I think of that before? 👍 for `A_COLUMN_HAS_NO_NAME`

![](http://i.giphy.com/69oDII4CxypTa.gif)